python/lsst/ap/pipe/ap_pipe.py

```diff
@@ -278,12 +307,16 @@ def _setupDatabase(configurable):
         A ConfigurableInstance with a database-managing class in its ``target``
         field. The API of ``target`` must expose a ``create_tables`` method
         taking no arguments.
+    Returns
+    -------
+    ppdb : `lsst.dax.ppdb.Ppdb`
+        Connection to an open and setup Ppdb.
     """
-    db = configurable.apply()
-    try:
-        db.create_tables()
-    finally:
-        db.close()
+    ppdb = daxPpdb.Ppdb(config=configurable,
+                        afw_schemas=dict(DiaObject=make_dia_object_schema(),
+                                         DiaSource=make_dia_source_schema()))
+    ppdb._schema.makeSchema()
```

Comment:
Given the discussion on lsst/dax_ppdb#5, this line should be `ppdb.makeSchema()` but otherwise doesn't need to be changed.

Reply:
So I should do both correct? (add the comment and change it to makeSchema).